plugin/federation/testdata/entityresolver/generated/federation.requires.go

```diff
@@ -0,0 +1,38 @@
+package generated
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/99designs/gqlgen/plugin/federation/testdata/entityresolver/generated/model"
+)
+
+// PopulateMultiHelloMultipleRequiresRequires is the requires populator for the MultiHelloMultipleRequires entity.
+func (ec *executionContext) PopulateMultiHelloMultipleRequiresRequires(ctx context.Context, entity *model.MultiHelloMultipleRequires, reps map[string]interface{}) error {
+	panic(fmt.Errorf("not implemented: PopulateMultiHelloMultipleRequiresRequires"))
+}
+
+// PopulateMultiHelloRequiresRequires is the requires populator for the MultiHelloRequires entity.
+func (ec *executionContext) PopulateMultiHelloRequiresRequires(ctx context.Context, entity *model.MultiHelloRequires, reps map[string]interface{}) error {
```

Comment:
`PopulateMultiHelloRequiresRequires` here is awkward and I'd like to avoid `RequiresRequires`

Reply:
Is MultiHelloRequires the entity name?